ETWAnalyzer/EventDump/DumpFile.cs

```diff
@@ -719,6 +727,7 @@ public string RootLevelDirectory
             public long FileSetSecurityCount { get; internal set; }
             public long FileDeleteCount { get; internal set; }
             public long FileRenameCount { get; internal set; }
+            public long FileOpenCloseTimeInus { get; internal set; }
```

Comment:
Do you need a new property here? It should suffice to define
public long FileOpenCloseTimeInus { get => FileOpenTimeInus+FileCloseTimeInus; }